config/webhook/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+- clientConfig:
+    caBundle: Cg==
```

Comment:
What does this do?

Reply:
This is generated by kubebuilder. I guess it's a required field. Maybe cert-manager or something else updates it? It decodes to `\n`.